app/assets/stylesheets/_color_variables.scss

```diff
@@ -0,0 +1,3 @@
+$color-title-main: white;
+$color-body: #E3DEC1;
+$color-buttons-main: #eee;
```

Comment:
is it ok if I do it later in the project? at this point I chose pretty much random colours just to setup the variables